code/modules/projectiles/gun_system.dm

```diff
@@ -228,6 +230,7 @@
 		QDEL_NULL(current_mag)
 	if(muzzle_flash)
 		QDEL_NULL(muzzle_flash)
+	QDEL_NULL(cell)
```

Comment:
no need to move this

Reply:
I don't even know if it's in energy gun code